solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -50,14 +50,23 @@ interface IRandomBeaconStaking {
         returns (uint256);
 }
 
+/// @title Random Beacon interface
+interface IRandomBeacon {
```

Comment:
👍

Reply:
I wasn't sure if this change to `random-beacon` can be included in this PR or it should go into a separate PR. These are separate projects in a monorepo.